src/pages/registration/description_tab/projects_field/ProjectContributorRow.tsx

```diff
@@ -58,24 +65,25 @@ export const ProjectContributorRow = ({ contributor }: ProjectContributorRowProp
 
   return (
     <>
-      <Box sx={{ display: 'grid', gridTemplateColumns: { xs: '1fr', sm: '1fr 2fr 3fr' }, gap: '0.5rem 1rem' }}>
-        <Field name="contributors[0].type">
-          {({ field, meta: { touched, error } }: FieldProps<string>) => (
+      <Box sx={{ display: 'grid', gridTemplateColumns: { xs: '1fr', sm: '150px 2fr 3fr' }, gap: '0.25rem 1rem' }}>
```

Comment:
liten luring med 150px her for å sikre at hele rollenavnet alltid vises, i tillegg til at griden ikke blir ujevn